test/Tests/SetOps.hs

```diff
@@ -41,7 +41,7 @@ setOpTests = testGroup "Set operations"
   , testProperty "invertInPlace middle"     prop_invertInPlace_middle
   , testProperty "invertInPlaceLong middle" prop_invertInPlaceLong_middle
 
-  , mkGroup "reverseBits" prop_reverseBits
+  , localOption (QuickCheckTests 500) $ mkGroup "reverseBits" prop_reverseBits
```

Comment:
It's better to use `adjustOption` with `max 500 n` to be able to increase number of tests via `--quickcheck-tests` even beyond 500.